cobbler/actions/buildiso/__init__.py

```diff
@@ -54,27 +82,40 @@ def __init__(self, api):
         self.distmap = {}
         self.distctr = 0
         self.logger = logging.getLogger()
-
-        self.profiles: List[str] = []
+        self.templar = templar.Templar(api)
         self.isolinuxdir = ""
 
+        # based on https://uefi.org/sites/default/files/resources/UEFI%20Spec%202.8B%20May%202020.pdf
+        # FIXME: PowerPC is missing from the table
```

Comment:
Is there a reason why we can't add it to the dict?

Reply:
I don't know what PowerPC uses for a known efi bootloader location. PowerPC is not listed in the linked UEFI spec.